cypress/integration/components/navbar/navbar.spec.js

```diff
@@ -0,0 +1,40 @@
+/// <reference types="cypress" />
```

Comment:
can remove this comment right?

Reply:
no actually that define that its a test file and opens other functionalities.